CMakeLists.txt

```diff
@@ -93,10 +110,12 @@ target_include_directories(${TARGET_NAME} PUBLIC)
 # TSKIT
 set(TARGET_NAME tables)
 file(GLOB_RECURSE TABLE_SOURCES ${PROJECT_SOURCE_DIR}/treerec/tskit/*.c)
+file(GLOB_RECURSE PARALLEL_TABLE_SOURCES ${PROJECT_SOURCE_DIR}/treerec/tskit/*.cpp)
 set(TSKIT_INCLUDES ${PROJECT_SOURCE_DIR}/treerec)
-add_library(${TARGET_NAME} STATIC ${TABLE_SOURCES})
+add_library(${TARGET_NAME} STATIC ${TABLE_SOURCES} ${PARALLEL_TABLE_SOURCES})
```

Comment:
Again, should only happen when the build is parallel; the cmake changes need to be completely isolated so that a standard build is unchanged

Reply:
Done!